crates/storage/provider/src/providers/database/provider.rs

```diff
@@ -726,6 +728,20 @@ impl<'this, TX: DbTx<'this>> AccountExtReader for DatabaseProvider<'this, TX> {
     }
 }
 
+impl<'this, TX: DbTx<'this>> AccountChangeReader for DatabaseProvider<'this, TX> {
+    fn account_changes_in_block(&self, block_number: BlockNumber) -> Result<Vec<AccountBeforeTx>> {
```

Comment:
Would rename it to `account_block_changeset` as changeset is the naming that we have.

Reply:
Done in 80810b804ebaea75aa500e1673c9c54bd2cc8b37